website/static/css/main.css

```diff
@@ -782,6 +782,12 @@ blockquote {
     display: none !important;
   }
 }
+@media (max-width: 500px) {
+  img.logo {
+    content:url("../img/Accord-Project-logo-small.svg");
+  }
+}
+
```

Comment:
Can you provide context for why this is added?

Reply:
ok removed it